tests/test_cache.py

```diff
@@ -31,6 +38,7 @@ def test_cache_delete_many(app, cache):
     assert cache.get("hi") is not None
 
 
+@pytest.mark.skipif(HAS_NOT_REDIS, reason="requires Redis")
 def test_cache_unlink(app, redis_server):
```

Comment:
Looked like this test was missing the `skipif(HAS_NOT_REDIS)` that the other Redis-requiring tests have, so I fixed that while I was in here.